src/service/service.js

```diff
@@ -1,3 +1,18 @@
-"use strict";
+'use strict';
 
-console.log(`Hello, world!`);
+const {Cli} = require(`./cli`);
+const {
+  DEFAULT_COMMAND,
+  USER_ARGV_INDEX,
+  ExitCode
+} = require(`../constants`);
+
+const userArguments = process.argv.slice(USER_ARGV_INDEX);
+const [userCommand] = userArguments;
+
+if (userArguments.length === 0 || !Cli[userCommand]) {
+  Cli[DEFAULT_COMMAND].run();
+  process.exit(ExitCode.success);
```

Comment:
Тут лучше поставить else вместо выхода с успешным кодом. Вообще в местах где выхода можно избежать корректным завершением программы так и следует делать

Reply:
```suggestion
if (userArguments.length === 0 || !Cli[userCommand]) {
  Cli[DEFAULT_COMMAND].run();
  process.exit(ExitCode.SUCCESS);
} else {
  Cli[userCommand].run(userArguments.slice(1));
}
```

не совсем понял, зачем избавляться от `process.exit(ExitCode.success);`?

что ты имеешь в виду под корректным завершением программы?

Reply:
Я имею ввиду завершение с кодом 0, всё таки эта операция жесткая и прерывает все асинхронные операции в фоне, что не всегда есть хорошо, поэтому если есть возможность мягкого завершения, то так и нужно делать